packages/react-scripts/template/src/components/example/UseZionAxios.js

```diff
@@ -0,0 +1,16 @@
+import { i18n } from '@fs/zion-locale'
+import { authHeaderInterceptor } from '@fs/zion-axios'
+import { useAxios } from 'use-axios-client'
+
+export default function useZionAxios(url) {
+  let config = {
+    headers: {
+      common: {},
+    },
+    url,
+  }
+  config.headers['Accept-Language'] = i18n.language
+  // We're using the authHeaderInterceptor from zion-axios to set up the config correctly:
+  config = authHeaderInterceptor(config)
+  return useAxios({ ...config })
```

Comment:
hmm, why can't we just take the instance from zion-axios and pass it to the use-axios hook? We've essentially copied the meaningful bits of zion-axios here, so it isn't providing much value any more. I was hoping it would be more transparent.

Reply:
Yeah that's probably a good idea.